packages/core/rollup.config.npm.mjs

```diff
@@ -0,0 +1,47 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs'
+import nodeResolve from '@rollup/plugin-node-resolve'
+import typescript from '@rollup/plugin-typescript'
+
+import createRollupConfig, { sharedOutput } from "../../.rollup/index.mjs"
+
+const plugins = [
+  nodeResolve({
+    browser: true
+  }),
+  commonjs(),
+  typescript(),
+  babel({ babelHelpers: 'bundled' }),
+]
+
+const external = [
+  // '@bugsnag/cuid',
```

Comment:
commented out code - is this being added back in for a subsequent PR or can this just be an empty array?

Reply:
see https://github.com/bugsnag/bugsnag-js/pull/2316/files#r1949315462